crates/core/src/node/zks.rs

```diff
@@ -139,7 +139,6 @@ impl InMemoryNode {
         if let Some(bytecode) = self.storage.load_factory_dep_alt(hash).await? {
             return Ok(Some(bytecode));
         }
-        println!("Nope");
```

Comment:
lol, this println sneaking through reminded me of below. IYKYK
![balatro-card](https://github.com/user-attachments/assets/b55e4e22-016a-4d39-bd9c-f17f326c7cf0)